lesson_5_hw.js

```diff
@@ -0,0 +1,68 @@
+/*
+1. 
+  Напиши функцію map(fn, array), яка приймає на вхід функцію та масив, 
+  та обробляє кожен елемент масиву цією функцією, повертаючи новий масив. 
+  Приклад:
+*/
+
+function map(fn,arr){
+  let newArr = [];
+  for(let i = 0; i < arr.length; i++){
+      newArr.push(square(arr[i]));
```

Comment:
```suggestion
      newArr.push(fn(arr[i]));
```
маєш обробляти функцією яка передається як аргумент